.github/workflows/tensorboards_web_application_test.yaml

```diff
@@ -17,14 +14,10 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v4
 
-    - name: Install KinD
-      run: ./tests/gh-actions/install_kind.sh
-
-    - name: Create KinD Cluster
-      run: kind create cluster --config tests/gh-actions/kind-cluster.yaml
-
-    - name: Install kustomize
-      run: ./tests/gh-actions/install_kustomize.sh
+    - name: Install KinD, Create KinD cluster and Install kustomize
+      run: |
+        chmod +x tests/gh-actions/install_KinD_create_KinD_cluster_install_kustomize.sh
```

Comment:
no chmod here, you need to fix the file in git itself

Reply:
I fixed this. If you have any concerns regarding this PR can you let me know? If not I can delete redundant files before the merge